docs/source/queries/simple.md

```diff
@@ -84,6 +84,19 @@ Each row can be parsed as a tuple of rust types using `rows_typed`:
 # async fn check_only_compiles(session: &Session) -> Result<(), Box<dyn Error>> {
 use scylla::IntoTypedRows;
 
+// NOTE: using unpaged queries for SELECTs is discouraged in general.
+// Query results may be so big that it is not preferable to fetch them all at once.
+// However, `query_unpaged` will return all results in one, possibly giant, piece
+// (unless a timeout occurs due to high load incurred by the cluster).
+// This:
+// - increases latency,
+// - has large memory footprint,
+// - puts high load on the cluster,
+// - is more likely to time out (because big work takes more time than little work,
+//   and returning one large piece of data is more work than returning one chunk of data).
+// To sum up, for SELECTs that may return return a lot of data prefer paged queries,
+// e.g. with `Session::query_iter()`.
```

Comment:
This comment should be changed as was the one in `docs/source/queries/result.md`

Reply:
You mean adding note about tombstones?

Reply:
Yes